src/types.ts

```diff
@@ -30,6 +34,20 @@ export interface Config {
   provider: ethers.providers.Provider;
 }
 
+export interface NetworkDomainMintableConfig {
+  znsHub: ZNSHub;
+  domainPurchaser: DomainPurchaserConfig;
+  services: {
+    apiClient: znsApiClient;
+  };
+}
+
+export interface DomainPurchaserConfig {
```

Comment:
Organization:
Echo of above;
This type is only ever used by the `actions/minting/mintNetworkDomain` action, so it should be defined and exported from there and not the top level `types.ts` file

Reply:
Updated